modules/connatixBidAdapter.js

```diff
@@ -111,11 +202,33 @@ export const spec = {
         mediaTypes,
         params,
         sizes,
+        adUnitCode
       } = bid;
+
+      let detectedViewabilityPercentage = null;
+
+      const element = document.getElementById(adUnitCode);
```

Comment:
@rares-mihai-preda let's add the container ID into params also, and use the `adUnitCode` from bid when we can't find the container provided by pub.
in this case we need to update the cnx bid adapter docs as well, with the new params